Library/Location.swift

```diff
@@ -102,19 +104,19 @@ struct Location {
     }
 
     private var today: Date {
-        return Calendar.autoupdatingCurrent.startOfDay(for: Date())
+        return Date()
     }
 
     private var tomorrow: Date {
-        return Calendar.autoupdatingCurrent.date(byAdding: .day, value: 1, to: self.today)!
+        return self.calendar.date(byAdding: .day, value: 1, to: self.today)!
     }
 
     private var yesterday: Date {
-        return Calendar.autoupdatingCurrent.date(byAdding: .day, value: -1, to: self.today)!
+        return self.calendar.date(byAdding: .day, value: -1, to: self.today)!
     }
 
     private func suntimes(forDate date: Date) -> SunCalc {
-        return SunCalc(date: date, timeZone: TimeZone.autoupdatingCurrent, latitude: self.coordinate.latitude, longitude: self.coordinate.longitude)
+        return SunCalc(date: date, timeZone: .current, latitude: self.coordinate.latitude, longitude: self.coordinate.longitude)
```

Comment:
`self.calendar`?

Reply:
Not sure what you mean.

Reply:
why pass in the calendar directly instead of using the `private let` that you've set up only in this method and not in any of the other methods?

Reply:
There's no calendar being passed here, just the timezone, no? 🤔

Reply:
*facepalm*

Any particular reason not to use `autoupdatingCurrent` to line up with the calendar, then?

Reply:
I always default to `.current` and not `.autoupdatingCurrent`, unless I feel I need auto-updating. Is that not the default? 🤔

Reply:
I use autoupdating current as the default since it'll always be current even if the user moves locations (ex, flies somewhere, or drives across time zones). And if we're gonna use autoupdating current for the calendar, we should definitely match that in the time zone - otherwise the calendar and the time zone may wind up thinking the user is in a different place.

Reply:
The thing is, it's never held on to. It's used when called, and discarded. I don't feel the need to add the extra "weight" of an auto-updating model, if it will never update. Even if you move timezones during the call, it would report pretty much the same, no? 🤔 I'm usually only using auto-updating if I hold on to it. For a "single-purpose" app that you open once, check and close for the day (usually), I don't see the need. That's one of the reasons I removed it in some other places. But if you feel that strongly about it, I can move it to auto-updating too. 👍

Reply:
TL;DR I was thinking of older APIs and this can stay.

The problem here is that I'm thinking of some of the implementation details around older APIs: `NSTimeZone` has a [bunch of weirdness](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/DatesAndTimes/Articles/dtTimeZones.html#//apple_ref/doc/uid/20000185-SW2) around [`default`](https://developer.apple.com/documentation/foundation/nstimezone/1387244-defaulttimezone) vs [`local`](https://developer.apple.com/documentation/foundation/nstimezone/1387209-localtimezone), mostly relating to `default` remaining cached under the hood if the user moves but the phone is not restarted, and `local` not. 

In the Swift-only APIs for `TimeZone` you just have `current` and [`autoupdatingCurrent`](https://developer.apple.com/documentation/foundation/timezone/2292819-autoupdatingcurrent), and there is no (documented) issue with the caching, so this can stay as is.

Reply:
Well, now I'm concerned about possible issues arising because of this. Any idea how we can test it?

Reply:
The easiest way is on device to change your system time zone and see if the time zone is updated correctly. I don't think there's a great way to test it automated since the behavior is under the hood by Apple APIs. 

Again, personally, I'd just throw it to `autoupdatingCurrent` just to make sure things are updated and call it a day.